libraries/lib-stretching-sequence/AudioSegmentSampleView.cpp

```diff
@@ -56,3 +56,16 @@ size_t AudioSegmentSampleView::DoCopy(float* buffer, size_t bufferSize) const
    }
    return written;
 }
+
+void FillBufferFromTrackBlockSequence(
+   const ChannelSampleView& track, float* buffer, size_t bufferSize)
+{
+   size_t written = 0;
+   for (const auto& segment : track)
```

Comment:
This moved function is now more general than it needs to be.  The first argument is always a size one vector at the sole place where you call it.  Maybe you feel like cleaning that up?